main.tf

```diff
@@ -164,6 +164,7 @@ resource "ibm_cos_bucket" "cos_bucket1" {
   resource_instance_id  = local.cos_instance_id
   region_location       = var.region
   cross_region_location = var.cross_region_location
+  endpoint_type         = var.bucket_endpoint
```

Comment:
bucket_endpoint seems misleading to a consumer, bucket_endpoint_type would be more meaningful (also because used to fill endpoint_type attribute)

Reply:
This was restored as it was before to match pre-existing usage.

If we want to change the name, maybe something like management_endpoint_type_for_bucket to try and highlight that it is not a bucket endpoint type, but a management endpoint type for buckets. Thoughts?

Reply:
As it has been restored to pre-existing usage I removed the request for changes.
However yes your proposal makes it more meaningful

Reply:
We can call it whatever we like now I guess since it was actually removed, so it wont be a breaking change. I'm easy on the naming, but I think the main thing we need to do is ensure the variable description is clear

Reply:
After various discussions, I will change the name of the variable so that is more obvious what it does.